.travis.yml

```diff
@@ -12,18 +8,15 @@ node_js:
   - "5"
   - node
 
-os:
-  - linux
+cache:
+  directories:
+    $HOME/.npm
+    $HOME/.cache # includes bowers cache
 
 before_install:
-  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
-  - test $TRAVIS_OS_NAME = "osx" && brew update && brew install phantomjs || test $TRAVIS_OS_NAME = "linux"
   - npm config set spin false
 
 install:
-  - node --version
```

Comment:
because TravisCI is showing them automatically already (see e.g. https://travis-ci.org/ember-cli/ember-cli/jobs/116771208#L151-L156)